Assignment/solution_student/CSS/style.css

```diff
@@ -0,0 +1,26 @@
+body{
+	margin-left: 60px;
+	margin-right: 60px;
+	margin-top: 60px;
+	margin-bottom: 60px;
+	background-color: white;
+	color: black;
+	text-align: center;
+	font-family: sans-serif;
+	border-top: 5px solid black;
+	border-bottom: 5px solid black;
+	padding-bottom: 30px;
+}
+h1{
+  font-size: 50px;
+  font-weight: bold;
+}
+strong {text-align: left;
+}
+footer{
+	padding-bottom: 1.5cm;
```

Comment:
It's quite unconventional to use `cm` as a unit of measurement on the web, so I would avoid it — I think it could be interpreted strangely. Instead us `px` for simple measurements like this one